boilerplate/lyft/github_workflows/pull_request.yml

```diff
@@ -4,47 +4,82 @@ on:
   pull_request
 
 jobs:
-  build-and-end2end:
+  build-docker:
```

Comment:
will we update boilerplate too, eventually?

Reply:
You mean the repo, right? yeah I've been trying to make the changes here that I can just copy paste into the repo... but so far this won't work directly... need more brain power